src/CoreBundle/private/modules/MTTableManager/MTTableManager.class.php

```diff
@@ -530,6 +542,7 @@ public function GetHtmlHeadIncludes()
         $includeLines = parent::GetHtmlHeadIncludes();
         $includeLines[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/jquery/flash/flash.js').'" type="text/javascript"></script>';
         $includeLines[] = '<link href="'.TGlobal::GetPathTheme().'/css/table.css" rel="stylesheet" type="text/css" />';
+        $includeLines[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/components/bootstrap3-typeahead/bootstrap3-typeahead.min.js').'" type="text/javascript"></script>';
```

Comment:
This is missing in the template engine includes.

Reply:
Moved it to the footer include file.